concent_api/conductor/tasks.py

```diff
@@ -123,6 +149,22 @@ def upload_acknowledged(
     )
     assert isinstance(subtask_id, str)
 
+    # Worker locks database row corresponding to the subtask in the subtask table.
+    if Subtask.objects.filter(subtask_id=subtask_id).exists():  # pylint: disable=no-member
```

Comment:
I think if we are starting tasks, Subtask corresponding with subtask_id should exist in our database. Now we have test `test_that_upload_acknowledged_task_should_log_error_when_verification_request_with_given_subtask_id_does_not_exist`, but `upload_acknowledged` is called in `upload_finished`. `upload_finished` is called in `blender_verification_request` or `report_upload`. I think we should be sure (by validation at the beginning of first celery task or just before call), that subtask exist in database.  It should be checked at the beginning, not in some function in the middle of flow. This problem does not concern this issue, so I decided to cover it like "easiest way", but I think it should be changed in separate issue